src/modules/fancyzones/lib/JsonHelpers.cpp

```diff
@@ -431,8 +433,8 @@ namespace JSONHelpers
 
             result.data.showSpacing = device.GetNamedBoolean(NonLocalizable::EditorShowSpacingStr);
             result.data.spacing = static_cast<int>(device.GetNamedNumber(NonLocalizable::EditorSpacingStr));
-            result.data.zoneCount = static_cast<int>(
-                device.GetNamedNumber(NonLocalizable::EditorZoneCountStr));
+            result.data.zoneCount = static_cast<int>(device.GetNamedNumber(NonLocalizable::EditorZoneCountStr));
+            result.data.sensitivityRadius = static_cast<int>(device.GetNamedNumber(NonLocalizable::EditorSensitivityRadiusStr, 20));
```

Comment:
Here would be better to add a constant. Let's create a namespace with constants in the `JsonHelpers.h` and add it there. Also, use this constant in the `FromJsonMissingSensitivityRadiusUsesDefault` test as the expected value.